common/prompt.ts

```diff
@@ -352,16 +359,19 @@ export function getPromptParts(opts: PromptPartsOptions, lines: string[], encode
   if (memory) parts.memory = memory.prompt
 
   const gaslight = opts.settings?.gaslight || defaultPresets.openai.gaslight
-  const ujb = opts.settings?.ultimeJailbreak
+  const ujb = opts.settings?.ignoreCharacterUjb
+    ? opts.settings?.ultimeJailbreak
+    : char.postHistoryInstructions || opts.settings?.ultimeJailbreak
+  const systemPrompt = opts.settings?.ignoreCharacterSystemPrompt
+    ? opts.settings?.systemPrompt
+    : char.systemPrompt || opts.settings?.systemPrompt
```

Comment:
Just realised. This uses `char` but should probably be `replyAs`. @sceuick correct?

Reply:
[198cb09](https://github.com/agnaistic/agnai/pull/401/commits/198cb09bd713832b396c29d2f69b9c363edc6f32)